homeassistant/components/climacell/config_flow.py

```diff
@@ -43,6 +56,7 @@ def _get_config_schema(
                 CONF_NAME, default=input_dict.get(CONF_NAME, DEFAULT_NAME)
             ): str,
             vol.Required(CONF_API_KEY, default=input_dict.get(CONF_API_KEY)): str,
+            vol.Required(CONF_API_VERSION, default=4): vol.In([3, 4]),
```

Comment:
Why does the user need to select api version? Is it not possible to just go with version 4?

Reply:
Yes I checked, they assign different keys for each version :/

Reply:
Let's do whatever you think is best here.